lecture_5/homework/app/services/reports/siege_report.rb

```diff
@@ -3,10 +3,23 @@
 module Reports
   class SiegeReport
     def initialize(building:)
+      @building = building
+      @warriors = building.warriors
     end
 
     def call
-      raise NotImprementedYet
+      @building.siege_ability = (@warriors.present? ? @building.granary / food_demand : 0)
```

Comment:
i mam pytanie co do wymagan: twierdza potrzebuje też ludzi, którzy nie są wojownikami. 
Czy to oznaczało, że powinienem stworzyć dodatkowy model np kucharz i uwzględnić go?